### ISSUE TITLE
[Suggestion] Make merge_stone option client-side rather than server-side

### ISSUE THREAD

**Issue by tanyan-tanyan** [FTBTeam/FTB-Ultimine#75]
```#Doesn't stop at different types of stones
merge_stone = false
```
Is it possible to make this config option client-side rather than server-side?
To my understanding, changing the client-side toml's merge_stone value to true doesn't make a difference if the server-side's merge_stone value is set to false, and vice versa.

It would be extremely useful if it was possible to switch between false and true value in-game or just not be restricted to the server's configuration, as our server's players have found the need for both of them, and right now they're limited to the server configuration.

In our modpack's case, we have a lot of new stone types which gets generated alongside vanilla stone. Some of our players wanted to have the ability to leave all stone intact except for the one stone type they want to mine, however when the value is set to true, they end up mining everything which is not convenient. 
On another hand, it is difficult to mine in small square mode if the value is set to false, if they're attempting to excavate large areas for builds.

Here's an example screenshot of the mess which happens in the latter scenario:
![javaw_WjOvTiVGEi](https://user-images.githubusercontent.com/68793071/110412960-f76baf00-8084-11eb-820e-3d7da5c2df55.png)

**Comment by RGFTheCoder** [comment]
Agreed; I had to restart my server to be able to get andesite for the create mod without getting a ton of extra cobblestone. Maybe there could be a type-based option (so you can say that they aren't merged in shapeless mode, but are merged in the rest of the modes) which also means players wouldn't have to change config.

**Comment by MaxNeedsSnacks** [comment]
Changed the title of the issue to reflect the current feature set, I'm currently debating whether adding "non-merging" variants of the various modes or adding a separate hotkey / config to toggle merging on and off, but either way, this'll require some restructuring

**Comment by HalJordan** [comment]
IMO the hotkey sounds like the better option, unless the config can be changed mid-game and seems like it would require less hard-coding than having to do a copy of each current shape just to get with and without merging.
